packages/experimental/Divider/SPEC.md

```diff
@@ -2,4 +2,161 @@
+
+Content, text or an icon are also not supported on mobile platforms.
+
+## Sample Code
+
+### Basic Divider
+
+```tsx
+<>
+  <Divider /> // This is a horizontal divider with no content.
+  <Divider insetSize={16} /> // Divider with a defined inset
+  <Divider vertical /> // Vertical divider
+  <Divider appearance="brand" /> // Divider with a set appearance
+</>
+```
+
+### Divider With Text
+
+```tsx
+<>
+  <Divider>Lorem ipsum</Divider> // Center-aligned horizontal divider
+  <Divider alignContent="start">Dolor sit</Divider> // Start-aligned horizontal divider
+  <Divider alignContent="end" vertical>
+    Amet consectetur
+  </Divider>{' '}
+  // End-aligned vertical divider
+</>
+```
+
+### Divider With Icon
+
+```tsx
+const svgIconProps = {...} // insert svg icon props here
+const fontIconProps = {...} // insert font icon props here
+
+<>
+  <Divider icon={{ svgSource: svgIconProps }} /> // With svg icon
+  <Divider icon={{ fontSource: fontIconProps }} /> // With font icon
+</>
+```
+
+## Variants
+
+### Vertical
+
+The `Divider` will be rendered as a vertical line, take up a minimum height of 84px, and not take up the entire row if the `vertical` prop is set to `true`.
+
+### With Content
+
+If text as passed as a child, or the icon prop is set, the `Divider` will render the content between the dividing line. It will also take up more width and height depending on the `Divider` orientation.
+
+#### Alignment
+
+Users can control where content appears in the `Divider`.
+
+- If `alignContent` is equal to "start", the content appears towards the left of the `Divider` if it's horizontal or towards the top if vertical.
+- If `alignContent` is equal to "end", the content appears towards the right of the `Divider` if it's horizontal or towards the bottom if vertical.
+- If `alignContent` is equal to "center" or not set, the content appears exactly in the middle of the dividing line.
+
+## API
+
+### Slots
+
+The `Divider` uses 6 total slots:
+
+- `root` [View] The outer container of the component
+- `beforeLine` [View] The dividing line rendered before passed content. If no content is passed, this is the only slot rendered.
+- `afterLine` [View] The dividing line rendered after passed content. Not rendered if no content is passed.
+- `wrapper` [View] If content is passed, this is the outer container of the content between `beforeLine` and `afterLine` which provides the correct padding for the content.
+- `text` [TextV1] If text is passed as a child, this renders inside `wrapper`.
+- `icon` [IconV1] If the `icon` prop is set, an icon will render inside `wrapper`. If text is also passed while the `icon` prop is set, then this slot will not render.
+
+These slots can be modified using the `compose` function on `Divider`. For more information on using the compose API, please see [this page](https://github.com/microsoft/fluentui-react-native/blob/main/packages/framework/composition/README.md).
```

Comment:
I think you have to expose the slots as props if you want them to be able to be modified. No compose on compressible, unfortunately